docs/src/rules/sort-keys.md

```diff
@@ -263,6 +264,127 @@ let obj = {
 
 :::
 
+### allowLineSeparatedGroups
+
+Examples of **incorrect** code for the `{allowLineSeparatedGroups: true}` option:
+
+::: incorrect
+
+```js
+/*eslint sort-keys: ["error", "asc", {allowLineSeparatedGroups: true}]*/
+/*eslint-env es6*/
+
+let obj1 = {
+    b: 1,
+    c () {
+
+    },
+    a: 3
+}
+
+let obj2 = {
+    b: 1,
+    c: 2,
+
+    z () {
+
+    },
+    y: 3
+}
+
+let obj3 = {
+    b: 1,
+    c: 2,
+    // comment
+    z () {
+
+    },
+    y: 3,
+}
```

Comment:
These two examples are basically the same. Maybe the comment in `obj3` should be between `z` and `y` to show that lines with comments do not count as lines that separate groups?

Reply:
yes, makes sense 👍🏻